rpm/cray/csm/noos/index.yaml

```
@@ -50,6 +50,7 @@ https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/noos/:
     - hpe-yq-4.33.3-1.aarch64
     - hpe-yq-4.33.3-1.x86_64
     - ilorest-4.2.0.0-20.x86_64
+    - iuf-cli-1.6.1-1.x86_64
```

Can you bump this to include my fix for [CASMINST-6518](https://jira-pro.it.hpe.com:8443/browse/CASMINST-6518)?

```suggestion
    - iuf-cli-1.6.2-1.x86_64
```